scipy-1.0/paper.tex

```diff
@@ -828,9 +830,8 @@ \subsection*{API and ABI evolution}
 significant) costs to users, \textit{and} only after giving clear deprecation
 warnings to those users for at least one year.  In general, we encourage
 changes that improve clarity in the API of the library but strongly discourage
-breaking backwards compatibility\footnote{
-    Backwards compatibility means that new versions of SciPy will keep user
-    code written for an older version working.}
+breaking backwards compatibility (Backwards compatibility means that new 
```

Comment:
Parentheses in the middle of a sentence usually don't include a full sentence, right? I think this would go after the sentence.  But then again, perhaps we can assume that the reader is familiar with the notion of backwards compatibility? It is not a technical term, really. And I'm sure there are loads of other places where familiarity with more technical stuff is assumed, so I'd just remove this.

Reply:
Agreed, removed.